.chloggen/698.yaml

```diff
@@ -0,0 +1,7 @@
+change_type: breaking
+
+component: messaging
```

Comment:
```suggestion
component: messaging, db
```

Either that (which I'm not sure it's a good idea..) or split the db changes into a separated PR.

I tried doing it the former locally, and it renders like this:

```
### 🛑 Breaking changes 🛑

- `messaging, db`: test (#123)
```

Reply:
ah, good point! 

I'll bring it up on db semconv and if there is support for this PR, I'll split this Pr in two. Otherwise, I'll close it.

I feel we should have the same decision for both cases and it wouldn't make sense to change messaging, but keep DB stuff.
Making it as draft for now